KDSocialShare is not abstract and does not override abstract method createJSModules(

Followed the guidelines to readme.

```
:react-native-social-share:compileReleaseJavaWithJavac - is not incremental (e.g. outputs have changed, no previous execution, etc.).
/Users/shrikantsonone/AskDOSS/doss.askdoss.homesearch.mobile/consumer_app/Homesearch/node_modules/react-native-social-share/android/src/main/java/com/barefootcoders/android/react/KDSocialShare/KDSocialShare.java:15: error: KDSocialShare is not abstract and does not override abstract method createJSModules() in ReactPackage
public class KDSocialShare implements ReactPackage {
       ^
1 error
:react-native-social-share:compileReleaseJavaWithJavac FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-social-share:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 19.397 secs
Could not install the app on the device, read the error above for details.
Make sure you have an Android emulator running or a device connected and have
set up your Android development environment:
https://facebook.github.io/react-native/docs/android-setup.html
```

--- a/android/src/main/java/com/barefootcoders/android/react/KDSocialShare/KDSocialShare.java
+++ b/android/src/main/java/com/barefootcoders/android/react/KDSocialShare/KDSocialShare.java
@@ -28,4 +28,9 @@ public class KDSocialShare implements ReactPackage {
     public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
         return Collections.emptyList();
     }
+
+    @Override
+    public List<Class<? extends JavaScriptModule>> createJSModules() {
+        return Collections.emptyList();
+    }
 }

